app/forms/proceeding_type_form.rb

```diff
@@ -0,0 +1,9 @@
+class ProceedingTypeForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  ATTRIBUTES = [:proceeding_type].freeze
+
+  attribute :proceeding_type
+  validates :proceeding_type, presence: true
```

Comment:
I think for completeness we should also validate `inclusion: { in: Flow::CaseDetailsHandler::PROCEEDING_TYPES }`, as we do that for other constrained options